.azure-pipelines/generation-templates/build-and-publish-typewriter.yml

```diff
@@ -16,7 +16,7 @@ steps:
 
 - task: CopyFiles@2
   inputs:
-    sourceFolder: '$(Build.SourcesDirectory)/src/Typewriter/bin/$(BuildConfiguration)/net5.0'
+    sourceFolder: '$(Build.SourcesDirectory)/src/Typewriter/bin/$(BuildConfiguration)/net6.0'
```

Comment:
Thanks for taking a look. I really appreciate it. 
I'll resolve this once I sort out the regression on the generator side so that I can bring back the build task.